README.md

```diff
@@ -92,8 +106,7 @@ Please note that if you have a GitHub account and want to push any work you migh
 up for future reference we suggest you [fork the repository](https://github.com/Cambridge-ICCS/ml-training-material/fork) 
```

Comment:
Sorry, I don't understand this sentence. Can it be simplified/split into two sentences?

Reply:
Reworded, does that look better?

Reply:
Yes, better